corehq/apps/app_manager/templates/app_manager/partials/add_property_button.html

```diff
@@ -11,13 +11,28 @@
       </button>
       <ul class="dropdown-menu">
           <li data-bind="click: addProperty"><a>{% trans "Property" %}</a></li>
+
           <!-- ko if: hqImport('app_manager/js/app_manager').checkCommcareVersion("2.17") -->
           <li data-bind="click: addGraph"><a>{% trans "Graph" %}</a></li>
           <!-- /ko -->
           <!-- ko ifnot: hqImport('app_manager/js/app_manager').checkCommcareVersion("2.17") -->
           <li class="disabled"><a>{% trans "Graph <small>(upgrade to 2.17 or greater)</small>" %}</a></li>
           <!-- /ko -->
+
+          <!-- ko if: hqImport("hqwebapp/js/initial_page_data").get("add_ons").calc_xpaths -->
+          <li data-bind="click: addXpathExpression"><a>{% trans "Calculated Property" %}</a></li>
+          <!-- /ko -->
+      </ul>
+    {% else %}
```

Comment:
Nit: it'd be DRYer to do `{% if request|toggle_enabled:'GRAPH_CREATION' or add_ons.calc_xpath %}` above instead of having the `else` block that repeats the caret and other markup. It's not terrible now, but it'll get more awkward when we add another optional item to this dropdown.